### PR TITLE
Adding ro-json

### PR DESCRIPTION
**Pull request by claydugo**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by claydugo** [comment]
@hmaarrfk this is ready for review!

File: recipes/ro-json/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,55 @@
+{% set name = "ro-json" %}
+{% set version = "1.0.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name | replace("-", "_") }}-{{ version }}.tar.gz
```
Comment (claydugo) [inline recipes/ro-json/meta.yaml:9]: This is how I dealt with the PyPI `-` aliasing. Please let me know if there is a better way.

**Comment by hmaarrfk** [comment]
See dependency trimming: https://github.com/ramonaoptics/ro-json/pull/2

Why did you have python's lower bound at 3.4 here and on pypi at 3.10?

File: recipes/ro-json/meta.yaml (hunk 1 above)
Comment (hmaarrfk) [reply recipes/ro-json/meta.yaml:10]: nice one

**Comment by claydugo** [comment]
>python's lower bound at 3.4 here 

Did that absentmindedly. Your change is an improvement.

**Comment by hmaarrfk** [comment]
not sure why windows is failing....

**Pull request merged**